gunicorn.conf.py

```diff
@@ -26,6 +26,10 @@
 
 
 def post_fork(server, worker):
+    # opentelemetry initialisation needs these env vars to be set, so ensure they are
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "airlock.settings")
```

Comment:
```suggestion
    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "jobserver.settings")
```

Reply:
:facepalm:

Reply:
FTR, this didn't fail because it was already set correctly